plugins/docsettingtweak.koplugin/main.lua

```diff
@@ -0,0 +1,107 @@
+local DataStorage = require("datastorage")
+local FFIUtil = require("ffi/util")
+local InputDialog = require("ui/widget/inputdialog")
+local LuaSettings = require("luasettings")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local _ = require("gettext")
+local T = FFIUtil.template
+local filemanagerutil = require("apps/filemanager/filemanagerutil")
+local util = require("util")
+
+local DocSettingTweak = WidgetContainer:new{
+    name = 'docsettingtweak',
+}
+
+local directory_defaults_name = "directory_defaults.lua"
+local directory_defaults_path = FFIUtil.joinPath(DataStorage:getSettingsDir(), directory_defaults_name)
+local directory_defaults = nil
+local initialized = false
+
+function DocSettingTweak:init()
+    if not initialized then
+        -- Make sure our settings file exists
+        if not lfs.attributes(directory_defaults_path, "mode") then
+            FFIUtil.copyFile(FFIUtil.joinPath(self.path, directory_defaults_name) .. ".template",
```

Comment:
This concatenation seems unnecessary. Clearer to just define above `local directory_defaults_template = "directory_defaults_template.lua"` (and put "template" before the extension, and keep it named something.lua so we don't lose Lua syntax highlighting.